tools/pidpersec.bt

```diff
@@ -4,8 +4,48 @@
  *		For Linux, uses bpftrace and eBPF.
  *
  * Written as a basic example of counting on an event.
- *
- * USAGE: pidpersec.bt
+ * 
+ * Example of usage:
+ * 
+ * # ./pidpersec.bt
+ * Attaching 4 probes...
+ * Tracing new processes... Hit Ctrl-C to end.
+ * 22:29:50 PIDs/sec: @: 121
+ * 22:29:51 PIDs/sec: @: 120
+ * 22:29:52 PIDs/sec: @: 122
+ * 22:29:53 PIDs/sec: @: 124
+ * 22:29:54 PIDs/sec: @: 123
+ * 22:29:55 PIDs/sec: @: 121
+ * 22:29:56 PIDs/sec: @: 121
+ * 22:29:57 PIDs/sec: @: 121
+ * 22:29:58 PIDs/sec: @: 49
+ * 22:29:59 PIDs/sec:
+ * 22:30:00 PIDs/sec:
+ * 22:30:01 PIDs/sec:
+ * 22:30:02 PIDs/sec:
```

Comment:
This could be shortened by few lines (the times and the below text would need to be updated, too).